crates/net/network/src/transactions/fetcher.rs

```diff
@@ -51,6 +52,9 @@ pub(super) struct TransactionFetcher {
     pub(super) filter_valid_hashes: AnnouncementFilter,
     /// Info on capacity of the transaction fetcher.
     info: TransactionFetcherInfo,
+    /// Soft limit for the byte size of a [`PooledTransactions`](reth_eth_wire::PooledTransactions)
+    /// response.
+    pub(super) soft_limit_byte_size_pooled_transactions_response: usize,
```

Comment:
please move `soft_limit_byte_size_pooled_transactions_response` into `TransactionFetcherInfo`

Reply:
u mean `soft_limit_byte_size_pooled_transactions_response_on_pack_request` right, also `soft_limit_byte_size_pooled_transactions_response_on_pack_request`'s default value is `SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE` which spec'd at 2 MiB? could u point me where it's spec'd?